package.json

```diff
@@ -33,5 +20,27 @@
     "Fabien O'Carroll <fabien@allou.is>",
     "Kirill Fomichev <fanatid@ya.ru>"
   ],
-  "license": "MIT"
+  "files": [
+    "index.js"
```

Comment:
When you have many-many modules and build application that save all files from package (electron for example) `files`, I think, is important. You can save a few megs (especially if some package uses big set of fixtures for tests). But partially I agree with yours words.